omero/developers/Python.rst

```diff
@@ -810,14 +810,8 @@ ROIs
     # Another helper for generating the color integers for shapes
     def rgba_to_int(red, green, blue, alpha=255):
         """ Return the color as an Integer in RGBA encoding """
-        r = red << 24
-        g = green << 16
-        b = blue << 8
-        a = alpha 
-        rgba_int = r+g+b+a
-        if (rgba_int > (2**31-1)):       # convert to signed 32-bit int
-            rgba_int = rgba_int - 2**32
-        return rgba_int
+        return int.from_bytes([red, green, blue, alpha],
+                          byteorder='big', signed=True)
```

Comment:
Is `signed` definitely correct?
```
In [12]: int.from_bytes([1],'big',signed=True)
Out[12]: 1

In [13]: int.from_bytes([255],'big',signed=True)
Out[13]: -1

In [14]: int.from_bytes([1],'big',signed=False)
Out[14]: 1

In [15]: int.from_bytes([255],'big',signed=False)
Out[15]: 255
```

Reply:
I transferred the code to the example from https://github.com/ome/openmicroscopy/pull/6235#discussion_r436163183

Reply:
Actually comparing the methods:
``int.from_bytes([255, 0, 0], byteorder='big')``
``16711680``
previous implementation
``-16776961``

Reply:
@manics This is how you need to invoke it 
``int.from_bytes([255, 0, 0, 255], 'big', signed=True)``
by default alpha is 0 not 255 as in our implementation
i.e.
``int.from_bytes([255, 0, 0, 255], 'big', signed=True)``
and ``rgba_to_int(255, 0, 0)``
returns the same result i.e. ``-16776961``

Reply:
I found the relevant documentation which explains why it's `signed`: https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome.html
Do you think it's helpful to include that link somewhere for developers?

Reply:
@manics I will add the link before the helper method